README.md

```diff
@@ -75,6 +75,28 @@ The RPM carries out a number of steps to run the SSM in a specific way.
 6. It creates the pidfile directory /var/run/apel/
 7. It installs a service script in /etc/init.d/
 
+## Installing the DEB
+
+### Installation
+Install APEL SSM.
```

Comment:
This and the following "Install..." lines should prob end in a colon as the code bit is a follow on.

Reply:
changed